src/pages/index.js

```diff
@@ -83,9 +83,9 @@ const IndexPage = ({ data }) => (
       <h3>Hello world!</h3>
     </Divider>
     <Divider justify={'space-around'}>
-      <Icon icon={ICONS.RULER} size={50} color={'white'} />
-      <Icon icon={ICONS.BLOG} size={50} color={'white'} />
-      <Icon icon={ICONS.RULER} size={50} color={'white'} />
+      <Icon icon={ICONS.RULER} size={50} color={props.theme.white} />
```

Comment:
👍

Reply:
:smile: 
I caught it via the browser console.